addons/hr_holidays/hr_holidays.py

```diff
@@ -231,7 +231,7 @@ def _check_date(self, cr, uid, ids, context=None):
     def name_get(self, cr, uid, ids, context=None):
         res = []
         for leave in self.browse(cr, uid, ids, context=context):
-            res.append((leave.id, leave.name or _("%s on %s") % (leave.employee_id.name, leave.holiday_status_id.name)))
+            res.append((leave.id, _("%s on %s : %.2f day(s)") % (leave.employee_id.name, leave.name or leave.holiday_status_id.name, leave.number_of_days_temp)))
```

Comment:
We had a task a few weeks ago to stop displaying the leave name and to display only the leave type name.